src/aap_eda/api/serializers/activation.py

```diff
@@ -215,6 +282,8 @@ class Meta:
             "git_hash",
             "status_message",
             "activation_id",
+            "event_stream_id",
+            "parent_type",
```

Comment:
I think parent_type is internal and we don't want to expose this field to the public api. Not even makes sense because it will be always the only same value.

Reply:
@Alex-Izquierdo We were having UI issues and were not sure if the UI should be checking for NULL values and would it need a type check.

Reply:
What are the issues? The UI should not need to do anything with the parent_type, it is internal. The endpoint should return the right type (activations in this case) For the UI, the type is implicit in the endpoint (activation_instances or event_stream_instances)